processes/third_party_vuln_process.md

```diff
@@ -77,7 +77,12 @@ At this point, a CVE should be requested through the HackerOne platform through
 [email](cve-assign@hackerone.com) that should include the Report ID and a summary.
 
 The vulnerability is considered as published when a Pull Request adding it
-to this repository is opened.
+to this repository is opened. Once the Pull Request is merged, and a new version
+is released, vulnerable versions must be deprecated using `npm deprecate`:
```

Comment:
s/must/should/, I don't think we can force module authors to do that.
Unless @npm automatically does that for us based on our data.